config/Run3_2022/samples.yaml

```diff
@@ -42,3 +36,1092 @@ MET_Run2022C:
+  sampleType: QCD
+QCD_HT100to200_MLM:
+  crossSection: QCD_HT100to200_MLM
+  generator: madgraphMLM
+  miniAOD: /QCD-4Jets_HT-100to200_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_HT200to400_MLM:
+  crossSection: QCD_HT200to400_MLM
+  generator: madgraphMLM
+  miniAOD: /QCD-4Jets_HT-200to400_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_HT400to600_MLM:
+  crossSection: QCD_HT400to600_MLM
+  generator: madgraphMLM
+  miniAOD: /QCD-4Jets_HT-400to600_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_HT600to800_MLM:
+  crossSection: QCD_HT600to800_MLM
+  generator: madgraphMLM
+  miniAOD: /QCD-4Jets_HT-600to800_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_HT800to1000_MLM:
+  crossSection: QCD_HT800to1000_MLM
+  generator: madgraphMLM
+  miniAOD: /QCD-4Jets_HT-800to1000_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_HT1000to1200_MLM:
+  crossSection: QCD_HT1000to1200_MLM
+  generator: madgraphMLM
+  miniAOD: /QCD-4Jets_HT-1000to1200_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_HT1200to1500_MLM:
+  crossSection: QCD_HT1200to1500_MLM
+  generator: madgraphMLM
+  miniAOD: /QCD-4Jets_HT-1200to1500_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_HT1500to2000_MLM:
+  crossSection: QCD_HT1500to2000_MLM
+  generator: madgraphMLM
+  miniAOD: /QCD-4Jets_HT-1500to2000_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_HT2000toInf_MLM:
+  crossSection: QCD_HT2000toInf_MLM
+  generator: madgraphMLM
+  miniAOD: /QCD-4Jets_HT-2000_TuneCP5_13p6TeV_madgraphMLM-pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+
+############## QCD_PT-*_EMEnriched ##############
+QCD_PT-10to30_EMEnriched:
+  crossSection: QCD_PT-10to30_EMEnriched
+  generator: pythia8
+  miniAOD: /QCD_PT-10to30_EMEnriched_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_PT-30to50_EMEnriched:
+  crossSection: QCD_PT-30to50_EMEnriched
+  generator: pythia8
+  miniAOD: /QCD_PT-30to50_EMEnriched_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_PT-50to80_EMEnriched:
+  crossSection: QCD_PT-50to80_EMEnriched
+  generator: pythia8
+  miniAOD: /QCD_PT-50to80_EMEnriched_TuneCP5_13p6TeV_pythia8/Run3Summer22MiniAODv4-130X_mcRun3_2022_realistic_v5-v2/MINIAODSIM
+  sampleType: QCD
+QCD_PT-80to120_EMEnriched:
```

Comment:
missing `sampleType`

Reply:
Oh this is the error from the previous copy/paste error